code/modules/mecha/equipment/weapons/mecha_ballistic.dm

```diff
@@ -0,0 +1,233 @@
+// Enhanced mecha gun with projectile storage // THE SLOPPA
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic
+	name = "\improper General Ballistic Weapon"
+//	step_delay = 20 // It can't be THAT heavy!
```

Comment:
Remove this and any other commented-out step_delays.

Reply:
Additionally - this is an undocumented change (you're buffing mech speed here). Add this to the OP and CL.

Reply:
I recommend noting down the step_delays in a .txt file on your computer if the idea is to remember what each is supposed to be